cmd/search/http.go

```diff
@@ -342,6 +350,62 @@ func renderSummary(ctx context.Context, w http.ResponseWriter, index *Index, gen
 	return count, err
 }
 
+func (o *options) handleSearch(w http.ResponseWriter, req *http.Request) {
+	index, err := o.parseRequest(req)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("Bad input: %v", err), http.StatusBadRequest)
+		return
+	}
+
+	if len(index.Search) == 0 {
+		http.Error(w, "The 'search' query parameter is required", http.StatusBadRequest)
+		return
+	}
+
+	result := map[string]map[string][]*Match{}
+
+	err = executeGrep(req.Context(), o.generator, index, 30, func(name string, search string, matches []bytes.Buffer, moreLines int) {
+		//metadata, ok := o.metadata.MetadataFor(name)
+		//if !ok {
+		//	panic(name)
+		//}
+		//fmt.Printf("%#+v\n", metadata)
+		// FIXME: convert 'name' to job-detail URI and asset URI
+		var ok bool
+
+		_, ok = result[name]
```

Comment:
trim leading slash here, I just added that in the other

Reply:
> trim leading slash here...

Fixed with 73e1d0a -> 26a014f.